pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -161,6 +162,8 @@ public class ExtensibleLoadManagerImpl implements ExtensibleLoadManager, BrokerS
     @Getter
     private final BrokerSelectionStrategy brokerSelectionStrategy;
 
+    private final BrokerSelectionStrategy roundRobinBrokerSelectionStrategy;
```

Comment:
nit: can we rename this to `sheddingExcludedNamespaceSelectionStrategy` in case we use another selection strategy in the future?